app/views/scores/index.html.erb

```diff
@@ -1,3 +1,4 @@
+<%= content_tag :div, '', id: 'score', data: { source: @score.id } %>
```

Comment:
This line at the top here allows me to grab this by it's id and the data attribute allows me to get the score's idea from the controller variable